src/Federation/One/ExternalDirectiveType.cs

```diff
@@ -1,7 +1,7 @@
 using ApolloGraphQL.Federation.HotChocolate.Constants;
 using ApolloGraphQL.Federation.HotChocolate.Properties;
 
-namespace ApolloGraphQL.Federation.HotChocolate;
+namespace ApolloGraphQL.Federation.HotChocolate.One;
```

Comment:
I think we should change the namespacing to `ApolloGraphQL.HotChocolate.Federation` otherwise its very odd package naming when supporting v1/v2 schemas....